.github/workflows/autobuild.yaml

```
@@ -109,28 +110,27 @@ jobs:
           git config --global user.email "actions@github.com"
           git add OpenTaiko/OpenTaiko.csproj
           git add CHANGELOG.md
-          git commit -m "Update changelog for version ${{steps.get-version.outputs.version}}"
+          git commit -m "Update changelog for version $env.version"

```

```suggestion
          git commit -m "Update changelog for version $env:version"
```
`$env.version` is invalid. It's `${{ env.version }}` for GitHub Workflow syntax or `$env.version` for Powershell syntax.